data/human/human missions.txt

```diff
@@ -6893,3 +6895,387 @@ mission "Lost Dog: Homeward Bound"
+				die
+
+			label "see us"
+			`	"I have no idea," says Bunker. "But it's probably time to go."`
+			choice
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we have to leave the drones behind."`
+
+			label "no can do"
+			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones, we'll lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
+			choice
+				`	"We're leaving. Now!"`
+				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
+					goto escape
+
+			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
+			choice
+				`	(Force him to stand down.)`
+					goto blaster
+				`	(Let him go to the cargo bay and lock down the drones.)`
+					goto escape
+
+			label "call drones"
+			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
+			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`
+
+			label escape
+			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`
+			`	You immediately raise the shields and take off. The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
+				launch
+
+			label blaster
+			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, tearing through the <ship>'s hull. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out from under you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
+				die
```

Comment:
I don't imagine it took you two minutes to pull out your blaster and restrain Bunker, so it seems odd to me that this choice would lead to the same outcome as deciding to stay and get more footage.

Perhaps we have an ending where you escape but don't save the drones, therefore causing different dialog in the next mission.

Reply:
I originally didn't consider this because I kinda wanted the player to progress... but then again, I said earlier I don't want to railroad people if they make the wrong decision. I'll make some changes soon.

(Also there's the conversation in the middle where he complains, so that's like at least 30 seconds of dialogue right????????)